scripts/kind-e2e/e2e.sh

```diff
@@ -17,11 +34,7 @@ function kind_clusters() {
             logs[$i]=$(mktemp)
             echo Creating cluster${i}, logging to ${logs[$i]}...
             (
-            if [[ -n ${version} ]]; then
-                kind create cluster --image=kindest/node:v${version} --name=cluster${i} --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
-            else
-                kind create cluster --name=cluster${i} --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
-            fi
+            kind create cluster --image=kindest/node:v${version} --name=cluster${i} --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
```

Comment:
This fails when `version` is set to `latest` (as the default value). Could you change it so that `version` includes the `v` when a version is specified? (Or any other appropriate fix!)

Reply:
@hunchback can you look at @skitt comment ^

Reply:
@skitt / @mangelajo - I brought back the old code